doc/guides/west/config.rst

```diff
@@ -165,14 +165,14 @@ extension commands (found in :file:`scripts/west_commands`).
    * - Option
      - Description
    * - ``build.pristine``
-     - String. Controls the way in which ``west build`` may run the ``pristine``
-       target before building. Can take the following values:
-
-         - ``never`` (default): Never automatically run the ``pristine`` target.
-         - ``auto``:  ``west build`` will automatically run the ``pristine``
-           target before building, if a build system is present and the build
-           will fail otherwise (e.g. the user has specified a different board or
-           application from the one previously used to make the build
+     - String. Controls the way in which ``west build`` may clean the build
```

Comment:
Mixing "clean" and "pristine" here when the build system has both targets is confusing.